src/Action/ViewPostAction.php

```diff
@@ -77,6 +85,17 @@ public function __invoke($permalink)
         ]);
     }
 
+    /**
+     * @param PostInterface $post
+     *
+     * @return bool
+     */
+    protected function isVisible(PostInterface $post): bool
```

Comment:
https://github.com/sonata-project/SonataNewsBundle/blob/05c76d919c0f92136a59e386b5fc5e55c0230a95/composer.json#L25

There are no return types in php 5.6